echo-notifications/src/test/groovy/com/netflix/spinnaker/echo/notification/ManualJudgmentTemplateTest.groovy

```diff
@@ -159,16 +160,17 @@ class ManualJudgmentTemplateTest extends Specification {
     rendered == expected
 
     where:
-
```

Comment:
Maybe the commit message could use a tweak since there are still some uses of `stripIndent` in the file?  Don't suppose you're fired up to convert this test to java?

Reply:
I'll give it a go.